Lombiq.NodeJs.Extensions/config/lombiq.markdownlint.json

```diff
@@ -2,6 +2,7 @@
   "default": true,
   "MD004": { "style": "dash" },
   "MD013": false,
+  "MD028": false,
```

Comment:
Reason for disabling is that it triggers false positive when you have two unrelated, consecutive block quotes next to each other. Which is perfectly normal in GitHub [with their info box system](https://github.com/orgs/community/discussions/16925).
More info in [this issue](https://github.com/DavidAnson/markdownlint/issues/263), which was basically closed because they won't do anything about it.